app/src/main/res/values/strings.xml

```diff
@@ -32,4 +32,6 @@
         The purpose of the game is to avoid obstacles, pass levels, earn achievements and have fun!
         \nTo make droid jump, tap on the left side of the screen.
         \nTo make droid crouch, tap and hold on the right side of the screen</string>
+    <string name="restart_text">restart</string>
```

Comment:
Should it be "Restart" and "Main menu"?

Reply:
No difference because all buttons are uppercase styled but better to be titled. Fixed.